code/datums/diseases/carpcough.dm

```diff
@@ -0,0 +1,44 @@
+/datum/disease/carpcough
+	name = "Carpcough"
+	form = "Infection"
+	max_stages = 4
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Carpotoxin"
```

Comment:
How are crew supposed to get this?

Reply:
At the moment, it is admin only as I thought it was too lethal for a level 7 biohazard. We could think about a more severe virus event that only includes actually deadly diseases. Would you be interested in something like that?

Reply:
I mean, how are crew supposed to get enough carpotoxin to treat this?

And yes, I'd like more dangerous disease events.

Reply:
Ah. Carp, when butchered, produce meat that contains some carpotoxin. So worst case, you wait for it to reach stage 4, then kill the carp and butcher it. You might have to try several times if the cure roll fails.

Reply:
You can also produce a nigh infinite amount of baby carps from the critter vendor in a fish tank.

Reply:
Also what mitch said, see
>Might also encourage people to farm some fish for carp toxin.